RubberduckTests/Grammar/VBAParserTests.cs

```diff
@@ -0,0 +1,142 @@
+        }
+
+        [TestMethod]
+        public void TestModuleHeader()
+        {
+            string code = @"VERSION 1.0 CLASS";
+            string expectedTree = @"
+(startRule
+    (module endOfStatement
+        (moduleHeader VERSION 1.0 CLASS)
+        endOfStatement
+        endOfStatement
+        endOfStatement
+        endOfStatement
+        endOfStatement)
+<EOF>)";
+            assertTree(code, expectedTree);
+        }
+
+        [TestMethod]
+        public void TestModuleConfig()
+        {
+            string code = @"
+BEGIN
+  MultiUse = -1  'True
+END";
+            string expectedTree = @"
+(startRule
+    (module
+        (endOfStatement
+            (endOfLine \r\n))
+        (moduleConfig BEGIN
+            (endOfStatement
+                (endOfLine \r\n ))
+            (moduleConfigElement
+                (ambiguousIdentifier MultiUse) = 
+                (literal -1)
+                (endOfStatement
+                    (endOfLine
+                        (comment 'True))
+                    (endOfLine \r\n)))
+        END)
+        endOfStatement
+        endOfStatement
+        endOfStatement
+        endOfStatement)
+<EOF>)";
+            assertTree(code, expectedTree);
+        }
+
+        [TestMethod]
+        public void TestDictionaryCallLineContinuation()
+        {
+            string code = @"
+Public Sub Test()
+    Set result = myObj _
+    ! _
+    call
+End Sub";
+            string expectedTree = @"
+(startRule
+    (module
+        (endOfStatement
+            (endOfLine \r\n))
+        endOfStatement
+        endOfStatement
+        endOfStatement
+        (moduleBody
+            (moduleBodyElement
+                (subStmt
+                    (visibility Public)
+                    Sub
+                    (ambiguousIdentifier Test)
+                    (argList ( ))
+                    (endOfStatement
+                        (endOfLine \r\n ))
+                    (block
+                        (blockStmt
+                            (setStmt
+                            Set
+                            (implicitCallStmt_InStmt
+                                (iCS_S_VariableOrProcedureCall
+                                    (ambiguousIdentifier result))) =
+                            (valueStmt 
+                                (implicitCallStmt_InStmt
+                                    (iCS_S_VariableOrProcedureCall
+                                        (ambiguousIdentifier myObj) _\r\n
+                                        (dictionaryCallStmt ! _\r\n
+                                            (ambiguousIdentifier
+                                            (ambiguousKeyword call))))))))
+                    (endOfStatement (endOfLine \r\n)))
+                    End Sub))
+                endOfStatement)
+            endOfStatement)
+<EOF>)";
+            assertTree(code, expectedTree);
+        }
+
+        private void assertTree(string code, string expectedTree)
```

Comment:
Ah yes of course, sorry, always get those mixed up.

The expectedTree might get really wrong and we're usually only interested in a small fragment. Should we change the assert to "contains" instead of "equals" instead? So it'll just check if e.g. the dictionarycall is existent and of the correct structure?